twitter/test/com/treode/twitter/server/handler/DrivesAttachHandlerSpec.scala

```diff
@@ -47,4 +48,21 @@ class DrivesAttachHandlerSpec extends FlatSpec with SpecTools {
         .statusCode (405)
       .when
         .get ("/")
+    }
+
+  it should "handle errors by returning a JSON array" in
+    served { case (port, controller) =>
+      val notification = Notification(AlreadyAttached("f1"), AlreadyAttached("f2"))
+      (controller.attach _) .expects (Seq.empty) .returning (supply (notification))
+      given
+        .port (port)
+        .body ("[]")
+      .expect
+        .statusCode (400)
+        .body (matchesJson ("""[
```

Comment:
The format should be

```[{message="..."}]```

This gives us a way to add more fields for each error later.